### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by pavedroad**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/.github/CONTRIBUTING.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

**Review by jdevcs — APPROVED** [review]

**Review by jdevcs — CHANGES_REQUESTED** [review]
pls also update test snapshots, so following tests are passing https://github.com/web3/web3.js/actions/runs/8243330205/job/22554409042?pr=6884#step:4:305

**Comment by jdevcs** [comment]
@pavedroad

**Comment by pavedroad** [comment]
> pls also update test snapshots, so following tests are passing https://github.com/web3/web3.js/actions/runs/8243330205/job/22554409042?pr=6884#step:4:305

Thanks!  

Modified.please review again.

**Comment by jdevcs** [comment]
> > pls also update test snapshots, so following tests are passing https://github.com/web3/web3.js/actions/runs/8243330205/job/22554409042?pr=6884#step:4:305
> 
> Thanks!
> 
> Modified.please review again.

re invoked CI, thanks

**Review by avkos — APPROVED** [review]

**Comment by jdevcs** [comment]
this will be done by our team member in diff PR/issue

File: packages/web3-errors/test/unit/__snapshots__/rpc-errors.test.ts.snap [hunk 1]
```diff
@@ -25,7 +25,7 @@ exports[`rpc errors EIP1193ProviderRpcError test constructor with un registered
   "cause": undefined,
   "code": 99999,
   "innerError": undefined,
-  "message": "An Rpc error has occured with a code of 99999",
```
Comment (jisen210) [inline packages/web3-errors/test/unit/__snapshots__/rpc-errors.test.ts.snap:28]: Occurred

**Comment by pavedroad** [comment]
Thanks!

**Pull request closed without merging**